src/Engine/Renderer/Texture/Texture.hpp

```diff
@@ -237,8 +237,9 @@ class RA_ENGINE_API Texture final {
      * @param gamma the gama value to use (sRGB is 2.4)
      * @note only 8 bit (GL_UNSIGNED_BYTE data format) textures are managed by this operator.
      */
+    // FIXME : replace the 2.4 constant by SRGB_GAMMA without requiring to include Eigen ...
     void sRGBToLinearRGB( uint8_t* texels, uint numCommponent, bool hasAlphaChannel,
-                          Scalar gamma = Scalar( 2.4 ) );
+                          Scalar gamma = 2.4_ra );
```

Comment:
Use value defined in Color.

Reply:
As mentioned in the PR summary, this will add a dependency on Texture over Eigen and then, increase the compilation time. If you think it is the better solution, let's go for that